content/nats-certificate-rotation.md

```diff
@@ -0,0 +1,216 @@
+# Rotating NATS Certificates
+
+The following strategy rotates the certificates and CA across the director, health monitor, NATS, and all VMs. See [Components of Bosh](bosh-components.md) for more information on core components.
+
+
+![image](images/nats_rotation.png)
```

Comment:
A first look, this image seems a little intimidating and a user probably wouldn't understand it. I do think it's helpful once they have a general understanding of the process and components, so maybe it's better at the end? Perhaps, to ensure they have a big picture summary at the start, we could instead have a bullet list with a super simple summary of steps? Something like...

1. Reconfigure VMs to trust old + new CA
1. Reconfigure director to use new CA
1. Reconfigure VMs to trust only new CA

Then the rest of the doc gets into the fun stuff which expands those conceptual steps into the implementation details.

Maybe?